subworkflows/local/ont_hq.nf

```diff
@@ -0,0 +1,16 @@
+include { PORECHOP_ABI as PORECHOP_ONT         } from '../../modules/nf-core/porechop/abi/main'
```

Comment:
No need to rename the module here, keep the original name please

Reply:
Also, no need for this to be a subworkflow if it's only one module

Reply:
I agree on the module name. The rest of the workflow is WIP, I just wrote the main structure in this version